plugins/chat/app/models/chat/user_chat_thread_membership.rb

```diff
@@ -8,6 +8,7 @@ class UserChatThreadMembership < ActiveRecord::Base
     belongs_to :last_read_message, class_name: "Chat::Message", optional: true
     belongs_to :thread, class_name: "Chat::Thread", foreign_key: :thread_id
 
+    attribute :thread_title_prompt, default: true
```

Comment:
do we usually do this?

Reply:
I think I added it as a temporary solution but forgot to solve the underlying issue. For some reason even though I had thread_title_prompt within the `BaseThreadMembershipSerializer` it was not picking it up within tests. I've updated the serializer to fall back to a false value if it doesn't respond to `object.thread_title_prompt`